userapi/storage/accounts/sqlite3/accounts_table.go

```diff
@@ -39,14 +40,16 @@ CREATE TABLE IF NOT EXISTS account_accounts (
     -- Identifies which application service this account belongs to, if any.
     appservice_id TEXT,
     -- If the account is currently active
-    is_deactivated BOOLEAN DEFAULT 0
+    is_deactivated BOOLEAN DEFAULT 0,
+	-- The account_type (user = 1, guest = 2, admin = 3, appservice = 4)
+	account_type INTEGER DEFAULT 2
```

Comment:
Please do not use defaults. They are error-prone. Dendrite should _always_ be setting this value correctly.